### PR TITLE
fix item page 

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
fix kit item + priceType
kit item was not enable on submit when kititem checkbox is checked 
price type was not taking a value using default index , because the value that we want to pass to priceType is the key

### CODE REVIEW

**Comment by imtithalhalalo** [comment]
Screenshot 2024-11-01 133220
when adding qty as zero

**Comment by imtithalhalalo** [comment]
Screenshot 2024-11-01 133309
when i don't add rows

**Review by imtithalhalalo — APPROVED** [review]

File: src/pages/iv-items/forms/ItemsForm.js [hunk 1]
```diff
@@ -196,18 +194,32 @@ export default function ItemsForm({ labels, maxAccess: access, setStore, store,
...
       formik.setFieldValue('ivtItem', false)
       formik.setFieldValue('trackBy', '')
       formik.setFieldValue('valuationMethod', '')
     } else {
       setOnKitItem(false)
+      formik.setFieldValue('priceType', '')
     }
   }, [formik.values.kitItem])
 
+  async function fetchfirstPriceType() {
```
Comment (NoraBitarSM) [inline src/pages/iv-items/forms/ItemsForm.js:210]: refused, we have a defaultIndex for a reason

**Review by NoraBitarSM — CHANGES_REQUESTED** [review]

File: src/pages/iv-items/forms/ItemsForm.js [hunk 2]
```diff
@@ -281,22 +283,21 @@ export default function ItemsForm({ labels, maxAccess: access, setStore, store,
...
                     values={formik.values}
+                    defaultIndex={onKitItem && 0}
                     name='priceType'
                     label={labels.priceType}
-                    defaultIndex={onKitItem ? 0 : null}
                     readOnly={formik.values.kitItem}
                     valueField='key'
                     displayField='value'
                     displayFieldWidth={1}
                     required={!formik.values.kitItem}
```
Comment (RebeccaFrem02) [inline src/pages/iv-items/forms/ItemsForm.js:293]: remove this leave it just required

File: src/pages/iv-items/forms/ItemsForm.js
Comment (RebeccaFrem02) [inline src/pages/iv-items/forms/ItemsForm.js:1]: image


after save and clear the window title shoul be removed

**Comment by RebeccaFrem02** [comment]
image

is currency here mandatory?

**Comment by RebeccaFrem02** [comment]
image

after submitting i got a new value in the barcode, check if this is the required behavior

**Comment by RebeccaFrem02** [comment]
image

**Comment by RebeccaFrem02** [comment]
image


check if standard cost here is required

**Review by RebeccaFrem02 — APPROVED** [review]

**Pull request merged**